### PR TITLE
Updates for latest probe-rs release

### PR DESCRIPTION
**Pull request by thefaxman**
* Update GDB calls
* Update for u64 address space changes

Signed-off-by: Ryan Fairfax <ryan@thefaxman.net>

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -18,10 +18,10 @@ ftdi = ["probe-rs/ftdi"]
 sentry = ["probe-rs-cli-util/sentry"]
 
 [dependencies]
-probe-rs = { version = "0.12.0", git = "https://github.com/probe-rs/probe-rs" }
```
[the remark this answers is not in the document]
Comment (thefaxman) [reply Cargo.toml]: Added the version back

**Review by Yatekii — APPROVED** [review]
Thank you!

bors r+

**Comment by thefaxman** [comment]
> Thank you!
> 
> bors r+

@Yatekii  - I've fixed the rustfmt failure - can you restart the workflow

**Comment by Yatekii** [comment]
bors retry

**Pull request merged**